cmd/junitreport/README.md

```diff
@@ -0,0 +1,31 @@
+# junitreport
+
+`junitreport` is a tool that allows for the consumption of test output in order to create jUnit XML.
+
+## Installation
+
+In order to build and install `junitreport`, from the root of the OpenShift Origin repository, run `hack/buil-go.sh cmd/junitreport`. 
+
+## Usage 
+
+`junitreport` can read the output of different types of tests. Specify which output is being read with `--type=<type>`. Supported test output types currently include `'gotest'`, for `go test` output. The default test type is `'gotest'`. 
+
+`junitreport` can output flat or nested test suites. To choose which type of output to use, set `--suites=<type>` to either `'flat'` or `'nested'`. The default suite output structure is `'flat'`. When creating nested test suites, `junitreport` will use `/` as the delimeter between suite names: `github.com/maintainer/repository/suite` will be parsed as a hierarchy of `github.com`, `github.com/maintainer`, *etc.* If you are requesting nested test suite output but do not want the root suite(s) to be as general as `github.com`, for example, set `--roots=<root suite names>` to be a comma-delimited list of the names of the suites you wish to use as roots. If the parser encounters a package outside of those roots, it will throw an error and fail.
```

Comment:
not sure I'd expect a failure for tests outside specified roots, more that the results would be ignored...